docs/attributes-registry/url.md

```diff
@@ -8,15 +8,37 @@ linkTitle: URL
 <!-- semconv registry.url(omit_requirement_level) -->
 | Attribute  | Type | Description  | Examples  |
 |---|---|---|---|
+| `url.domain` | string | Domain of the url, such as "opentelemetry.io". [1] | `www.foo.bar`; `opentelemetry.io`; `3.12.167.2`; `[1080:0:0:0:8:800:200C:417A]` |
```

Comment:
How do these relate to [server.address](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/server.md) and [client.address](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/client.md)?

Reply:
Similar answer as for the `port`. I think it has no semantic correlation with server/client but represents extracted data from provided url for future use by the user.